g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationWithPatternAction.java

```diff
@@ -52,12 +52,20 @@
 	   public APIResponseDTO getTransPattern(TranslationWithPatternDTO data) throws Exception {
 	      logger.info("begin getTransPattern");
 	      if (validate(data)) {
+	    	  List<String> availableVersions = null;
+	          try {
+	        	  availableVersions = productService.getSupportVersionList(data.getProductName());
+	          }catch(L3APIException e) {
+	        	  availableVersions = null;
```

Comment:
availableVersions is null if exception happen here, so this line is extra and please remove it.

Reply:
done